ArduPlane/mode_takeoff.cpp

```diff
@@ -152,6 +149,13 @@ void ModeTakeoff::update()
         plane.calc_nav_pitch();
         plane.calc_throttle();
         //check if in long failsafe, if it is recall long failsafe now to get fs action via events call
```

Comment:
This comment belongs with the bit of code under the addition.

Reply:
will fix